### PR TITLE
fix: remove dash from python dist and module name

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Fixes https://github.com/hashicorp/cdktf-provider-googlebeta/issues/30

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -53,8 +53,14 @@ export class CdktfProviderProject extends cdk.JsiiProject {
         name: `@${namespace}/provider-${providerName}`,
       },
       python: {
-        distName: `${namespace}-cdktf-provider-${providerName}`,
-        module: `${namespace}_cdktf_provider_${providerName}`,
+        distName: `${namespace}-cdktf-provider-${providerName.replace(
+          /-/gi,
+          "_"
```
Comment (ansgarm) [inline src/index.ts:58]: Why underscore and not hyphen?

Comment (DanielMSchmidt) [reply src/index.ts:58]: because the hypen is the problem and not supported in pypi as far as I can tell

Comment (DanielMSchmidt) [reply src/index.ts:58]: This is the original issue, where google-beta gets auto underscored by someone in the process, but only for the dist info 

```
cdktf_cdktf_provider_google-beta
cdktf_cdktf_provider_google_beta-0.0.27.dist-info
```

Comment (ansgarm) [reply src/index.ts:58]: Ah, I see! Python dependency management always confused me^^

**Review by ansgarm — APPROVED** [review]

**Pull request merged**